server/remote_cache/digest/digest.go

```diff
@@ -51,6 +51,14 @@ func init() {
 		digestType repb.DigestFunction_Value
 		sizeBytes  int
 	}{
+		{
+			digestType: repb.DigestFunction_SHA512,
+			sizeBytes:  sha512.Size,
+		},
+		{
+			digestType: repb.DigestFunction_SHA384,
+			sizeBytes:  sha512.Size384,
+		},
```

Comment:
I don't think this is sufficient. We probably gonna need an update in `HashForDigestType` to link the digest function to the actual implementation of Go's `hash.Hash` interface.